packages/suite/src/views/wallet/transactions/components/TransactionList/components/TransactionItem/components/Target/index.tsx

```diff
@@ -62,26 +63,31 @@ interface TargetProps {
     isLast?: boolean;
     accountKey: string;
     targetMetadata?: string;
+    accountMetadata?: AccountMetadata;
 }
 
 export const Target = ({
     target,
     transaction,
     targetMetadata,
+    accountMetadata,
```

Comment:
this is probably unnecessary props pass-through. There is `targetMetadata` which may  be inferred from `accountMetadata` at lower level.